app/workers/zync_worker.rb

```diff
@@ -9,7 +9,7 @@ class ZyncWorker
   class_attribute :publisher
   self.publisher = ->(*args) { Rails.application.config.event_store.publish_event(*args) }
 
-  sidekiq_options retry: 3, dead: false, queue: :low
+  sidekiq_options retry: 3, dead: false, queue: :zync
```

Comment:
Why creating a new queue? Can't we just control the priority of the `low` queue?

Reply:
@guicassolato Because there is no need to limit the whole `low` queue to what Zync can take (which is veeeeery little, like 5 at once or so).